srml/staking/src/lib.rs

```diff
@@ -892,6 +900,14 @@ decl_module! {
 		fn set_invulnerables(validators: Vec<T::AccountId>) {
 			<Invulnerables<T>>::put(validators);
 		}
+
+		/// Add reward points to validator.
+		///
+		/// At the end of the era each the total payout will be distributed among validator
+		/// relatively to their points.
+		fn add_reward_points_to_validators(add: Vec<(T::AccountId, u32)>) {
+			<CurrentEraRewards<T>>::append(&add[..]).unwrap();
```

Comment:
this probably ain't so good - we should probably accumulate them in place to avoid huge storage entries.